docs/whatsnew_1.5.7.html

```diff
@@ -173,16 +172,20 @@ <h2>
                 The most notable improvements and bugfixes in v1.5.7 since v1.5.6 include:
             </h2>
             <ul>
-                <li>Core - Updated hundreds of files for compatibility with PHP 7.2, 7.3 and 7.4.  <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
-                <li>Admin - Default notification behavior for order update (send email/do not send email/hide update) may now be set</li>
-                <li>Admin - Admins can now login to customer accounts to place orders</li>
-                <li>Admin - Added product-details popover to allow quick visibility into products bought in an order without opening the order</li>
+                <li>Core - Many files have been updated for compatibility with PHP 7.2, 7.3 and 7.4. <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
+                <li>Admin - The default notification behavior for an Order Status update (send email/do not send email/hide update) may now set by an admin switch</li>
+                <li>Admin - Order Status emails are now sent in the language of the order</li>
+                <li>Admin - Admins can now login to a customer account (login as that customer), to place an order on their behalf</li>
+                <li>Admin - Order Listing. An order-product-details popover column has been added to allow a quick view of the products ordered without opening the order</li>
                 <li>Admin - Validation for integer min and max values and email addresses</li>
-                <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
-                <li>Admin - Order status updatable from order listing screen</li>
```

Comment:
Was removing this "quick update of order's status" intentional?

Reply:
no